src/db.js

```diff
@@ -1,18 +1,18 @@
 const flatfile = require('flat-file-db')
 const promise = require('promise')
 
-const db = flatfile.sync(process.env.DB_NAME || 'views.db')
+let adapter
 
-const promisifiedDb = {
-  // Promisify async operations
-  put: promise.denodeify(db.put.bind(db)),
-  del: promise.denodeify(db.del.bind(db)),
-  clear: promise.denodeify(db.clear.bind(db)),
-
-  get: db.get.bind(db),
-  has: db.has.bind(db),
-  keys: db.keys.bind(db),
-  close: db.close.bind(db),
+if (process.env.DB_ADAPTER) {
+  adapter = require('micro-analytics-adapter-' + process.env.DB_ADAPTER)
```

Comment:
We need to `try/catch` this part and make sure users get a descriptive warning message if the require fails since this is a bit magic. Something like:

```JS
try { /* ... */ }
catch (err) {
  // No idea if/how we can check that this is a module not found error
  if (isModuleNotFoundError(err)) {
    console.error(`You specified "${process.env.DB_ADAPTER}" as the DB_ADAPTER, but no package called "micro-analytics-adapter-${process.env.DB_ADAPTER}" was found. Please make sure you spelled the name correctly and have npm installed the necessary adapter package!`)
  }
}
```